site/docs/v1.3.0-beta.1/customize-installation.md

```diff
@@ -12,6 +12,14 @@
   - [Generate YAML only](#generate-yaml-only)
   - [Additional options](#additional-options)
   - [Optional Velero CLI configurations](#optional-velero-cli-configurations)
+    - [Enabling shell autocompletion](#enabling-shell-autocompletion)
+  - [Bash on Linux](#bash-on-linux)
```

Comment:
I think "Bash on Linux", "Bash on macOS" and "Autocompletion on Zsh" entries should be nested under "Optional Velero CLI configurations"

Reply:
thanks for catching that. The toc was auto generated based on the markdown headers. fixed the headers.